lib/lighthouse/audit_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'json'
+
+# Compares a url's actual score to the expected score.
+class AuditService
+  def initialize(url, audit, score)
+    @url = url
+    @audit = audit
+    @score = score
```

Comment:
Could `@score` default to `Lighthouse::Matchers.minimum_score`?

Reply:
I figured that the `AuditService` should just receive a score, whatever it is, and not care about whether it is custom or default.

Reply:
I guess I just looked at the settings being assigned below and thought that it comes from the same place

Reply:
It was because both the message and the audit needed it, so it seemed to make sense to to keep it out of the service.